src/solc_0.7/common/Interfaces/IERC1155.sol

```diff
@@ -6,7 +6,7 @@ pragma solidity 0.7.1;
     @dev See https://eips.ethereum.org/EIPS/eip-1155
```

Comment:
I wonder if we should use openzepelin interfaces when we can. IERC1155 is supported by openzeppelin
Same for IERC1155Receiver